backend/tests/conftest.py

```
@@ -0,0 +1,605 @@
+mock_fastapi.Depends.side_effect = lambda x: x
+mock_fastapi.HTTPException = HTTPException
+mock_fastapi.FastAPI = MagicMock()
+
+# Mock FastAPI websockets
+mock_websockets = MagicMock()
+mock_websockets.WebSocket = MagicMock()
+mock_websockets.WebSocketDisconnect = Exception
+mock_websockets.WebSocketState = MagicMock()
+
+# Mock FastAPI testclient
+class MockResponse:
```

Ruff

Redefinition of unused `MockResponse` from line 73

(F811)